dandi/tests/test_organize.py

```diff
@@ -256,3 +257,42 @@ def error_link(src: Any, dest: Any) -> NoReturn:
     monkeypatch.setattr(os, "symlink", succeed_link if sym_success else error_link)
     monkeypatch.setattr(os, "link", succeed_link if hard_success else error_link)
     assert detect_link_type(tmp_path) == result
+
+
+@pytest.mark.parametrize("mode", ["copy", "move"])
+@pytest.mark.parametrize("video_mode", ["copy", "move", "symlink", "hardlink"])
+def test_video_organize(video_mode, mode, create_video_nwbfiles):
+    dandi_organize_path = create_video_nwbfiles.parent / "dandi_organized"
+    cmd = [
+        "--files-mode",
+        mode,
+        "--modify-external-file-fields",
+        "--media-files-mode",
+        video_mode,
+        "-d",
+        str(dandi_organize_path),
+        str(create_video_nwbfiles),
+    ]
+    video_files_list = list((create_video_nwbfiles.parent / "video_files").iterdir())
+    video_files_organized = []
+    r = CliRunner().invoke(organize, cmd)
+    assert r.exit_code == 0
+    for nwbfile_name in dandi_organize_path.glob("**/*.nwb"):
+        vid_folder = nwbfile_name.with_suffix("")
+        assert vid_folder.exists()
+        with NWBHDF5IO(str(nwbfile_name), "r", load_namespaces=True) as io:
+            nwbfile = io.read()
+            # get iamgeseries objects as dict(id=object_id, external_files=[])
+            ext_file_objects = _get_image_series(nwbfile)
+            for ext_file_ob in ext_file_objects:
+                for no, name in enumerate(ext_file_ob["external_files"]):
+                    video_files_organized.append(name)
+                    # check if external_file arguments are correctly named according to convention:
+                    filename = Path(
+                        f"{vid_folder.name}/{ext_file_ob['id']}_external_file_{no}"
+                    )
+                    assert str(filename) == str(Path(name).with_suffix(""))
```

Comment:
There's no need to cast both sides to a `str`.  `Path` objects can be compared for equality just fine.